### PR TITLE
schema change

### PR DESCRIPTION
**Pull request by Li-Evelyn**
changing the assignment, question, and submission entities to use id/assignment_id for referencing assignments instead of the original course_id and assignment_name - should improve performance and make handling a bit easier

### CODE REVIEW

**Review by TaliaWang — APPROVED** [review]
yay it lookin ✨good✨ to me

File: server/query.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ populate_queries = [ // see populate_tables.sql
     `COPY Course (name) FROM '${process.env.CSV_PATH}course.csv' DELIMITER ',' CSV HEADER`,
     `COPY Teaches (course_id, staff_id) FROM '${process.env.CSV_PATH}teaches.csv' DELIMITER ',' CSV HEADER`,
     `COPY EnrolledIn (course_id, student_id) FROM '${process.env.CSV_PATH}enrolledin.csv' DELIMITER ',' CSV HEADER`,
-    `COPY Assignment (course_id, name, deadline, max_grade) FROM '${process.env.CSV_PATH}assignment.csv' DELIMITER ',' CSV HEADER`,
-    `COPY Question (course_id, assignment_name, number, max_grade, description) FROM '${process.env.CSV_PATH}question.csv' DELIMITER ',' CSV HEADER`,
-    `COPY QuestionSubmission (student_id, course_id, assignment_name, question_number, file_path, grade, staff_comments) FROM '${process.env.CSV_PATH}question_submission.csv' DELIMITER ',' CSV HEADER`,
-    `COPY AssignmentSubmission (student_id, course_id, assignment_name, grade, is_submitted) FROM '${process.env.CSV_PATH}assignment_submission.csv' DELIMITER ',' CSV HEADER`    
+    `COPY Assignment (course_id, name, deadline, max_grade, description) FROM '${process.env.CSV_PATH}assignment.csv' DELIMITER ',' CSV HEADER`,
```
Comment (pwang14) [inline server/query.js:28]: Our production data doesn't seem to currently have description, but I didn't seem to run into any issues importing the data (production is just NULL).

Comment (pwang14) [reply server/query.js:28]: LTGM tho!

**Review by pwang14 — APPROVED** [review]

File: server/query.js (hunk 1 above)
Comment (Li-Evelyn) [reply server/query.js:28]: max added descriptions to the prod data so it should be good now

**Pull request merged**